target/linux/ath79/dts/qca9563_tplink_archer-c7-v5.dts

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9563_tplink_archer-x7-v5.dtsi"
+
+/ {
+	compatible = "tplink,archer-c7-v5", "qca,qca9563";
+	model = "TP-Link Archer C7 v5";
+};
+
+&mtdparts {
+	factory-uboot@0 {
+		label = "factory-uboot";
+		reg = <0x000000 0x020000>;
+		read-only;
+	};
+
+	uboot: u-boot@20000 {
+		label = "u-boot";
+		reg = <0x020000 0x020000>;
+		read-only;
+	};
+
+	partition@40000 {
+		label = "partition-table";
+		reg = <0x040000 0x010000>;
+	};
+
+	art: art@50000 {
+		label = "art";
+		reg = <0x050000 0x010000>;
+		read-only;
+	};
+
+	info: info@60000 {
+		label = "info";
+		reg = <0x060000 0x020000>;
+	};
+
+	user_config: user-config@80000 {
+		label = "user-config";
+		reg = <0x080000 0x040000>;
+	};
+
+	firmware@c0000 {
+		label = "firmware";
+		reg = <0x0c0000 0xf00000>;
+		compatible = "denx,uimage";
+	};
+
+	default_config: default-config@ff0000 {
```

Comment:
same as above

We also do not need the "default_config" label for the node as only the `info` and `art` partition are referenced in the included dtsi.